src/centos-stream/manifest.json

```diff
@@ -42,6 +42,18 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/centos-stream/10.0/amd64",
+              "os": "linux",
+              "osVersion": "centos-stream10",
+              "tags": {
+                "centos-stream-10": {}
```

Comment:
The [updated tagging policy](https://github.com/dotnet/dotnet-buildtools-prereqs-docker?tab=readme-ov-file#how-to-identify-an-image) says that tag names are to explicitly define the architecture.

Reply:
Fixed with 8c562c1c1762105e67eef14582fe05631b9e19ac